purchase_stock_price_unit_sync/models/purchase_order.py

```diff
@@ -17,9 +17,9 @@ def write(self, vals):
 
     def stock_price_unit_sync(self):
         for line in self.filtered(lambda l: l.state in ["purchase", "done"]):
-            line.move_ids.write(
+            line.move_ids.mapped("stock_valuation_layer_ids").write(
```

Comment:
Is it correct to write al these records at once?

Reply:
Yes!! in base module product_avco_cost_sync is recomputed all items..
ping @carlosdauden